### ISSUE TITLE
Callbacks Potentially Undefined

### ISSUE THREAD

**Issue by jfrey** [ocetnik/react-native-background-timer#34]
I'm hitting an error on react-native-background-timer/index.js:58:17 in Android and iOS:

`undefined is not an object (evaluating 'this.callbacks[a]={callback:e,interval:!0,timeout:t}')`

I'm still looking for a consistent set of steps to reproduce as the timers are working properly for some of my users and not for others.

I'm on 1.0.3 currently and will update if I can track down more specifics.

Thanks!

**Comment by jfrey** [comment]
Going back to 1.0.2 our use case seems to work correctly so I'll look into the delta between versions.

**Comment by ocetnik** [comment]
Hi @jfrey,

do you still have this problem?

**Comment by jfrey** [comment]
Yes, I've just pinned to a version before the class change. If I set an interval from the background (geolocation update for example) this occurs but it did not occur in the prior function based approach.

________________________________
